jboss/container/util/pkg-update/execute.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+for mgr in yum dnf microdnf; do
+    if test -x "/usr/bin/$mgr"; then
+        mgr="/usr/bin/$mgr"
+        break
+    fi
+done
+if ! test -x "$mgr"; then
```

Comment:
Have you considered initializing `mgr` to say, `not-exist`, and test against that here? Like `if [ "$mgr" == "not-exist"]; then ...`